### PR TITLE
 Run tests on the new LTS 18.04

### PR DESCRIPTION
**Pull request by alkino**
* The new Ubuntu LTS is 18.04, dump the version inside CI

### CODE REVIEW

File: .github/workflows/coverage.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ env:
 
 jobs:
   coverage:
-    runs-on: ubuntu-16.04
+    runs-on: ubuntu-18.04
```
Comment (pramodk) [inline .github/workflows/coverage.yml:20]: hopefully no package name changes!

**Review by pramodk — APPROVED** [review]
LGTM

File: .github/workflows/coverage.yml (hunk 1 above)
Comment (alkino) [reply .github/workflows/coverage.yml:20]: it's why this PR exists: it's a try :p

Comment (pramodk) [reply .github/workflows/coverage.yml:20]: Docker is the way Nico! I quickly boot docker Ubuntu 18.04 image and see package name. You save some CI minutes 😎

Comment (alexsavulescu) [reply .github/workflows/coverage.yml:20]: Not so eco-friendly heh

Comment (pramodk) [reply .github/workflows/coverage.yml:20]: @alkino  : `libfl-dev` needs to be installed on Ubuntu 18.04

Comment (alkino) [reply .github/workflows/coverage.yml:20]: Yes, I already have been trapped inside nmodl for that

**Pull request merged**